client/app/shared/custom-dropdown/custom-dropdown.component.spec.js

```diff
@@ -0,0 +1,41 @@
+describe('Custom Dropdown component - ', function () {
```

Comment:
don't hate me for knit picking, convention throughout tests so far has been `Component: component-name`